ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -1464,6 +1466,34 @@ public void shouldFailIfCreateAsSelectExistingSourceTable() {
     makeSingleRequest(createSql, CommandStatusEntity.class);
   }
 
+  @Test
+  public void shouldInlineRunScriptStatements() {
```

Comment:
Shall we add a test with more than 1 statement?

Reply:
Nar, not for this functionality.  As I mentioned in the description of the PR the majority of use of this functionality will only have a single statement.  Multi-statement will work, but ROI on extensive testing is pretty low IMHO.